client/src/modules/reports/generate/inventory_report/inventory_report.config.js

```diff
@@ -0,0 +1,95 @@
+angular.module('bhima.controllers')
+  .controller('inventory_reportController', InventoryReportConfigController);
+
+InventoryReportConfigController.$inject = [
```

Comment:
I can't find where you use `$timeout` in this module.  Could you make sure you clean out all unused dependencies?  They make it harder to understand what the code is supposed to do.